### PR TITLE
Implement banana cake local state 

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch implements banana cake local state to get/set value from the
local state. It accesses the local state using the JNI Bridge.

Refs: #888

### CODE REVIEW

**Comment by jdragonbae** [comment]
@romandev, @codeimpl This is draft PR. I'm not sure whether I'm right. PTAL.

File: banana_cake/java/org/banana/cake/CakeLocalState.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import org.banana.cake.interfaces.BananaLocalState;
+
+import org.chromium.base.annotations.NativeMethods;
+import org.chromium.chrome.browser.preferences.Pref;
+
+public class CakeLocalState implements BananaLocalState {
+    @Override
+    public boolean isSecureDNSMode() {
+        return !CakeLocalStateJni.get().getString(Pref.DNS_OVER_HTTPS_MODE).equals("off");
```
Comment (romandev) [inline banana_cake/java/org/banana/cake/CakeLocalState.java:18]: I think you could just use `SecureDnsBridge.getMode();`, couldn't it?

Comment (jdragonbae) [reply banana_cake/java/org/banana/cake/CakeLocalState.java:18]: @romandev Thanks for the review. 

> I think you could just use SecureDnsBridge.getMode();, couldn't it?

Yes, we could use `SecureDnsBridge.getMode()` with access modifier change about `SecureDnsBridge`.
I'll try to change and upload it.

Comment (jdragonbae) [reply banana_cake/java/org/banana/cake/CakeLocalState.java:18]: I uploaded the new patch[1].

[1] https://github.com/triplebanana/triple_banana/pull/901

**Pull request closed without merging**